homeworks/olya.kosenko_olyabot/homework_2/Task_1.js

```diff
@@ -0,0 +1,33 @@
+function pyramid(n1) {
+    console.log(`Pyramid with ${n1} rows:`);
+    const str1 = '#';
+    const str2 = ' ';
+    for (let i = 0; i < n1; i++) {
+        console.log(str2.repeat(n1 - i) + str1.repeat(i * 2 + 4));
+    }
+}
+
+function check(n1) {
+    if (!Number.isNaN(n1) && n1 > 0) {
```

Comment:
nice, thank you for improving but does this `if` statement really necessary, or it can be replaced with just
`return condition`?